provider/app.py

```diff
@@ -82,7 +82,7 @@ def deleteTrigger(namespace, trigger):
     return response
 
 
-@app.route('/ping')
+@app.route('/')
 def testRoute():
     return jsonify('pong')
```

Comment:
Not sure if "pong" is the appropriate message to display since you are removing "ping" from the URL.